src/@next/components/atoms/Checkbox/Checkbox.tsx

```diff
@@ -7,6 +7,7 @@ const ENTER_KEY: number = 13;
 const SPACE_KEY: number = 32;
 
 export const Checkbox: React.FC<IProps> = ({
+  dataCy,
```

Comment:
Isn't there a better name for this attribute? What if we switched from Cypress to another testing framework, should we need to refactor the entire codebase?

Reply:
Good point

Reply:
Ill go with `data-test`, its also one of blessed names by cypress

Reply:
I'm not talking about its name in the resulting DOM, I'm talking about the prop name we pass around in React code. We should really name it after its function rather than after whatever it ends up being called in the browser.